maven/jdk17/Dockerfile.nanoserver

```
@@ -1,14 +1,16 @@
 # escape=`
-# this JAVA version is the one used for the execution of the agent process and must match the controller version
-ARG JAVA_VERSION=11.0.17_8
+ARG JAVA_VERSION=17.0.5_8
 FROM eclipse-temurin:${JAVA_VERSION}-jdk-windowsservercore-1809 AS core
-FROM jenkins/inbound-agent:jdk17-nanoserver-1809
+# Retrieve JDK11 from inbound-agent for running jenkins agent process but do not use it as default (this JAVA version is the one used for the execution of the agent process and must match the controller version)
+FROM jenkins/inbound-agent:jdk11-nanoserver-1809
```

```suggestion
FROM jenkins/inbound-agent:3077.vd69cf116da_6f-3-jdk11-nanoserver-1809
```

https://hub.docker.com/layers/jenkins/inbound-agent/3077.vd69cf116da_6f-3-jdk11-nanoserver-1809/images/sha256-74f62682e2544f835a72a8e427b2c09cd635ce385187663de3d1c3e7451d7aa2?context=explore